contents/docs/surveys/display-conditions.mdx

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I renamed this whole file; do I need to change anything about the URL routing for this to resolve?

Reply:
I think this is an old file, it's not linked anywhere, and is out of date, we can delete it.

Reply:
It is already being redirected https://posthog.com/docs/surveys/targeting

Reply:
oh okay word.